packages/slice-machine/src/features/customTypes/customTypesBuilder/GenerateSliceWithAiModal/GenerateSliceWithAiModal.tsx

```diff
@@ -307,5 +314,5 @@ async function addSlices(newSlices: NewSlice[]) {
     ),
   );
 
-  return slices;
+  return { slices, libraryName: library };
```

Comment:
why not just library (it's just a string)
```suggestion
  return { slices, library };
```

Reply:
Done ☑️